gvr-x3d-demo/app/src/main/java/org/gearvrf/x3ddemo/X3DparserScript.java

```diff
@@ -70,21 +55,13 @@
  *
  */
 
-public class X3DparserScript extends GVRScript
+public class X3DparserScript extends GVRMain
```

Comment:
Yes, I was think that made sense to rename it:
X3DParserMain.java to be consistent with other demo’s.

Glad you mentioned it.

From: rahul [mailto:notifications@github.com]
Sent: Tuesday, September 6, 2016 5:20 PM
To: gearvrf/GearVRf-Demos <GearVRf-Demos@noreply.github.com>
Cc: Mitchell Williams <m1.williams@partner.samsung.com>; Author <author@noreply.github.com>
Subject: Re: [gearvrf/GearVRf-Demos] Using GVRMain in place of GVRScene to be consistent (#315)


In gvr-x3d-demo/app/src/main/java/org/gearvrf/x3ddemo/X3DparserScript.java<https://github.com/gearvrf/GearVRf-Demos/pull/315#discussion_r77742390>:

> @@ -70,21 +55,13 @@

>   *

>   */

>

> -public class X3DparserScript extends GVRScript

> +public class X3DparserScript extends GVRMain

You might want to rename this call to X3DparserMain since it is no longer a script.

You could do it in a future pull request.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/gearvrf/GearVRf-Demos/pull/315/files/38d354c004d3b43036d7b826be060c1ae86cce20#r77742390>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ARSfvns9clyS2JdkZ0XNC0cTPvSP9X0Qks5qngNHgaJpZM4J2TH9>.